packages/client/src/authentication/Authentication.spec.ts

```diff
@@ -1,19 +1,89 @@
+import { DateUtils } from '@lens-protocol/shared-kernel';
+
 import { mumbaiSandbox } from '../consts/environments';
+import { CredentialsExpiredError, NotAuthenticatedError } from '../consts/errors';
 import { Authentication } from './Authentication';
+import { setupRandomAuthentication } from './__helpers__/setupAuthentication';
```

Comment:
`setupRandomAuthentication` means using `ethers.Wallet.random()` and not the one from env var. I'm trying to promote the usage of random wallets in tests and deprecate the need to share private key.

Reply:
I see, I think the name is somewhat confusing, hence my question. Maybe we can think of something more clear as we iterate over other features.

Reply:
we have a ticket to rethink testing strategy, let's do it there https://avara.height.app/lens-sdk/T-6018